web_theme_social/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+{
+'name': 'Odoo.Social Theme',
+'author': 'Rossa S.A.',
+'category': 'Hidden',
+'version': '8.0.1.0',
+'description':
+"""
+Odoo.Social web theme
+========================
+This modules changes the web interface colors and and brand
+""",
+'depends': [
+	'web',
+],
+'data' : [
+	'theme.xml',
+	'views/webclient_templates.xml'],
+'qweb' : [
+    "static/src/xml/*.xml",
+],
+'test': [],
+'installable': True,
+'auto_install': True,
```

Comment:
please, set auto_install to False. 

Or explain why you think it's necessary :
With this setting any people that use OCA / Web will have this module installed. Is it expected ?